src/qibolab/result.py

```diff
@@ -61,8 +32,11 @@ def q(self):
         return self.array.q
 
     def __add__(self, data):
```

Comment:
I am not sure if you have any use case in mind for this, but it is generally fine. Just wanted to point out that as it is now, if `self` or `data` also contain shots, these will be ignored in the object returned by `__add__`.

Reply:
This was mainly done for the live-plotting, but I think that in general this could be useful and easy to maintain.